examples/httpserver_websocket.py

```diff
@@ -0,0 +1,82 @@
+# SPDX-FileCopyrightText: 2023 Dan Halbert for Adafruit Industries
```

Comment:
Are you open to having your own name or github handle for the copyright notice in the new files that you've written. Or perhaps even added to the library components that you've made lots of modifications too?

Reply:
Of course. I was not available during your live, but I watched the video save after and i saw the DJDevon3's comment on how to specify the name in these lines. Is my understanding that it should be `# SPDX-FileCopyrightText: 2023 XYZ ` for non-employees correct? How about when one employee and one non-employee are to be specified?

Reply:
As far as I am aware it is allowed to have comma separated lists of names for the copyright in the event that there are multiple authors

i.e.:
```
# SPDX-FileCopyrightText: 2023 Dan Halbert for Adafruit Industries, Michal Pokusa
```